Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/PrefabFocusHandler.cpp

```diff
@@ -149,12 +150,30 @@ namespace AzToolsFramework::Prefab
             auto editUndo = aznew PrefabFocusUndo("Focus Prefab");
             editUndo->Capture(entityId);
             editUndo->SetParent(undoBatch.GetUndoBatch());
-            FocusOnPrefabInstanceOwningEntityId(entityId);
+
+            if (focusBehavior == FocusChangeBehavior::CloseCurrentlyFocusedItems)
+            {
+                FocusOnPrefabInstanceOwningEntityId(entityId);
+            }
+            else
+            {
+                FocusOnPrefabInstanceOwningEntityIdWithoutClosingCurrent(entityId);
+            }
         }
 
         return AZ::Success();
     }
 
+    PrefabFocusOperationResult PrefabFocusHandler::FocusOnParentOfFocusedPrefab([[maybe_unused]] AzFramework::EntityContextId entityContextId)
+    {
+        return FocusOnParentOfFocusedPrefab(entityContextId, FocusChangeBehavior::CloseCurrentlyFocusedItems);
+    }
+
+    PrefabFocusOperationResult PrefabFocusHandler::FocusOnParentOfFocusedPrefabWithoutClosingCurrent(AzFramework::EntityContextId entityContextId)
+    {
+        return FocusOnParentOfFocusedPrefab(entityContextId, FocusChangeBehavior::IgnoreCurrentlyFocusedItems);
+    }
```

Comment:
I'm confused on why we need these helper functions, since both the caller and the underlying `FocusOnParentOfFocusedPrefab` both use the `FocusChangeBehavior` enum to track the behavior.